src/site/_includes/js/eligibility.js

```diff
@@ -545,6 +545,10 @@ function onHouseholdMemberAdd(event) {
   // Add listener to the new member's spouse checkbox
   const spouseInput = newMember.querySelector('[id^="hh-member-spouse"]');
   spouseInput.addEventListener('click', onChangeSpouse);
+  spouseInput.dependentParameter = newMember.querySelector('[id^="hh-member-dependent"]');
```

Comment:
If you move this line after line 549, you can do

```
spouseInput.dependentParameter = dependentInput;
```

Reply:
Done; see latest commit ([e2377b4](https://github.com/theunitedeffort/theunitedeffort.org/pull/462/commits/e2377b44e3bfdb462234d02091db479f7d2dfb0e)).